index.html

```diff
@@ -11,272 +11,227 @@
+            </article>
+            <article class="work-item">
+              <h1>Profesional Art Printing Data</h1>
+              <p class="inter">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="work-stack">
+                <li class="stack-item">html</li>
+                <li class="stack-item">bootstrap</li>
+                <li class="stack-item">Ruby</li>
+              </ul>
+              <a href="" class="btn-a">See Project</a>
+            </article>
+            <article class="work-item">
+              <h1>Profesional Art Printing Data</h1>
+              <p class="inter">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="work-stack">
+                <li class="stack-item">html</li>
+                <li class="stack-item">bootstrap</li>
+                <li class="stack-item">Ruby</li>
+              </ul>
+              <a href="" class="btn-a">See Project</a>
+            </article>
+          </div>
+        </section>
+        <section class="full-h bkg-black" id="about-me">
+          <div class="flex-all">
+            <h1 class="crete">About Me</h1>
+            <p class="poppins">
+              I can help you build a product, feature or website. Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don't hesitate to contact me.
+            </p>
+            <a href="" class="btn-a w-part">Get My Resume</a>
+          </div>
+          <hr class="pad-right" />
+        </section>
+        <section class="bkg-black" id="skills-wrapper">
+          <div class="flex-all-c" id="skills">
+            <div class="skills-group inter">
+              <ul class="skills-list">
+                <li class="inai"><span class="sq sq-45"></span>Language</li>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+              </ul>
+            </div>
+            <div class="skills-group">
+              <ul class="skills-list">
+                <li class="inai"><span class="sq"></span>Frameworks</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+              </ul>
+            </div>
+            <div class="skills-group">
+              <ul class="skills-list">
+                <li class="inai"><span class="sq circle"></span>Skills</li>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
```

Comment:
- [ ] Kindly add the right `font-family` to the elements of the 2nd and 3rd list like you did with the 1st one 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/212333014-61be8664-c232-438a-8133-85374dc90abf.png)
![image](https://user-images.githubusercontent.com/50721479/212333105-12218584-1836-41e9-ba67-2328ef285507.png)


## Figma design:
![image](https://user-images.githubusercontent.com/50721479/212333069-cd71b932-fa2b-4c46-91c9-ef239f07560a.png)
![image](https://user-images.githubusercontent.com/50721479/212333143-d24711a9-9bd1-481b-b60e-15632137c062.png)